.sparse/makepkg-git

```diff
@@ -52,33 +52,16 @@
 /usr/share/makepkg/
 /usr/bin/zstd.exe
 
-# AsciiDoctor
-/mingw64/bin/asciidoctor
-/mingw64/bin/ruby.exe
-/mingw64/bin/x64-msvcrt-ruby*.dll
-/mingw64/lib/ruby/*/x64-mingw32/rbconfig.rb
-/mingw64/lib/ruby/*/rubygems.rb
-/mingw64/lib/ruby/*/rubygems/
-/mingw64/lib/ruby/*/delegate.rb
-/mingw64/lib/ruby/*/uri.rb
-/mingw64/lib/ruby/*/uri/
-/mingw64/lib/ruby/*/x64-mingw32/stringio.so
-/mingw64/lib/ruby/*/monitor.rb
-/mingw64/lib/ruby/*/x64-mingw32/monitor.so
-/mingw64/lib/ruby/gems/*/gems/asciidoctor-*
-/mingw64/lib/ruby/gems/*/specifications/asciidoctor-*.gemspec
-/mingw64/lib/ruby/*/tsort.rb
-/mingw64/lib/ruby/*/set.rb
-/mingw64/lib/ruby/*/logger.rb
-/mingw64/lib/ruby/*/x64-mingw32/enc/encdb.so
-/mingw64/lib/ruby/*/x64-mingw32/enc/windows_1252.so
-/mingw64/lib/ruby/*/optparse.rb
-/mingw64/lib/ruby/*/x64-mingw32/strscan.so
-/mingw64/lib/ruby/*/pathname.rb
-/mingw64/lib/ruby/*/x64-mingw32/pathname.so
-/mingw64/lib/ruby/*/did_you_mean*
-/mingw64/lib/ruby/*/cgi/util*
-/mingw64/lib/ruby/*/logger*
+# AsciiDoc
+/clangarm64/bin/asciidoc.exe
+/clangarm64/bin/asciidoc-script.py
+/clangarm64/bin/python*
+/clangarm64/bin/libpython*.dll
+/clangarm64/lib/python*/
+!/clangarm64/lib/python*/site-packages/
+!/clangarm64/lib/python*/test/
+/clangarm64/lib/python*/site-packages/asciidoc/
+/clangarm64/lib/python*/site-packages/asciidoc*-info/
```

Comment:
I tried to be more mindful here by including only specific folders from `/clangarm64/lib/python`, but the amount of folders became so large that I just chose to include the entire folder instead and exclude the largest unnecessary ones. With this config, the entire `/clangarm64/lib/python3.10` folder is ~26MB, so I think that's ok.

Reply:
Sure. An alternative might be to use `asciidoctor` instead, which uses Ruby, like we use to build `mingw-w64-x86_64-git-doc-html`...

Reply:
I'd love to, but `asciidoctor` [isn't available yet](https://github.com/git-for-windows/MINGW-packages/blob/147af4445ca6af015b88dd87a466591d88324e2a/mingw-w64-git/PKGBUILD#L101) for `clang-aarch64` 😅